Installer/Package.wxs

```diff
@@ -0,0 +1,42 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs"
+     xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+
+  <Package Name="pbench"
```

Comment:
It would be nice to support both per-user and per-machine installation, but I think it could be done separately.